lambda_functions/rds_init/requirements.txt

```diff
@@ -0,0 +1,2 @@
+aws-lambda-powertools==2.8.0
+psycopg2-binary==2.9.5
```

Comment:
For security reasons we might want to [use hashes](https://github.com/jazzband/pip-tools#using-hashes) here.

We probably also want a Dependabot configuration for this file.

Reply:
I will have a think about how to manage packages and dependencies for lambda. Probably a separate PR, assuming there is a better way of managing / handling this.

Reply:
Added hashes.